tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -564,7 +695,12 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
        |}
        |""".stripMargin,
     "def exist@@",
-    """def exist: Set[_] = ${0:???}""".stripMargin
+    """def exist: Set[_] = ${0:???}""".stripMargin,
+    filter = (str) => str.contains("def"), // for Scala3
```

Comment:
Why do we needthe filter here?

Reply:
Otherwise, we get additional completion items like

```
 CompletionItem [
  label = "existentials - scala.languageFeature"
  kind = Module
  tags = SeqWrapper ()
  detail = " scala.languageFeature"
  ...
]
```

that comes from compiler completions. 
This issue has a same root cause as https://github.com/scalameta/metals/pull/3897#discussion_r872040704, I'll leave a comment why

Reply:
We don't need it anymore thanks to https://github.com/scalameta/metals/pull/3897#discussion_r875491408